### PR TITLE
System architecture documentation

### PR DESCRIPTION
**Pull request by LunaAsefaw**
1. Image attachments upload
2. System architecture documentation
3. Data model upload (output from schema spy)

### CODE REVIEW

File: docs/src/developer/guide/architecture.md [hunk 1]
```diff
@@ -1,10 +1,42 @@
 # System Architecture
-<!-- Replace all of the titles with relevant titles -->
+In this section, we outline the system architecture using ER Diagrams, Software Component Diagrams etc. and key libraries / frameworks used in this project.
 
 ## Frameworks Used
```
Comment (Jeremy-Prior) [inline docs/src/developer/guide/architecture.md:4]: Please refer to the [documentation naming conventions](https://kartoza.github.io/sawps/developer/documentation/#file-naming-conventions) for heading conventions.

File: docs/src/developer/guide/architecture.md [hunk 2]
```diff
@@ -1,10 +1,42 @@
...
 
+The following is a list, with brief descriptions, of the key components used in creating this platform. Please refer to their individual documentation for in-depth technical information.
+
+| Logo | Name | Notes |
+|------------|---------|----------------|
+|![Django](img/arch-framework-django-1.png){: style="height:30px;width:30px"} | [Django](https://djangoproject.com) | Django makes it easier to build better web apps more quickly and with less code. |
```
Comment (Jeremy-Prior) [inline docs/src/developer/guide/architecture.md:10]: Please update the image names to something like:

```suggestion
|![Django](img/architecture-django-1.png){: style="height:30px;width:30px"} | [Django](https://djangoproject.com) | Django makes it easier to build better web apps more quickly and with less code. | 
```

And remove any images you are no longer using

**Review by Jeremy-Prior — APPROVED** [review]
The docs LGTM

@dimasciput will need to provide input on the data model site

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 1 min 18 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 27 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 26 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 26 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 1 min 8 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 24 min

**Comment by dimasciput** [comment]
Will merge this for now, we can update it later

**Review by dimasciput — APPROVED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 22 min

**Pull request merged**